inc/header-image-interviewees.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This is the template the loads the relevant hero image.
```

Comment:
It is different, it does not use the post thumbnail as the first choice of image. if it would use this, then the thumbnail of the first post would be used as the hero image 
The interviewees page uses a different breadcrumbs file already so I followed this convention since it is not a real wordpress 'page'

Reply:
That difference makes sense, thanks. Can this statement itself be changed to reflect that distinction? Right now both `header-image-interviewees.php` and `header-image.php` use this statement, without any differentiation in their operation.